Claudia/src/main/java/com/endava/task/second/OnlineShop.java

```diff
@@ -0,0 +1,60 @@
+package com.endava.task.second;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class OnlineShop {
+    public static void main(String[] args) {
+        List<Order> myOrders = new ArrayList<>();
+        List<User> userList = new ArrayList<>();
+        User user1 = new User("Jhon", "Home", new Address("Moldova", "Chisinau", "Mircea cel Batran"));
+        User user2 = new User("Catalina", "Office", new Address("Moldova", "Cahul", "Eliade 32"));
+
+        user1.addAddress("Billing", new Address("Moldova", "Orhei", "Mircea Eliade 3"));
```

Comment:
Your user1.addAddress() method isn't adding anything to user because you are creating new HashMap in this method and never adding any entry to already existing one address (which was created via constructor)
I.e. you never update addressMap variable.
Could you please change this logic in order to avoid this situation.

Reply:
Done.